CarpetX/src/interpolate.cxx

```diff
@@ -850,6 +889,7 @@ extern "C" void CarpetX_Interpolate(const CCTK_POINTER_TO_CONST cctkGH_,
   // Set result
   CCTK_REAL *const restrict *const restrict resultptrs =
       static_cast<CCTK_REAL *const *>(resultptrs_);
+  assert(int(recvbuf.size()) == (nvars + 1) * npoints);
```

Comment:
`assert` is too late the code in `CCTK_DEBUG` in line 876 alreay touches all points.

Reply:
This `assert` can trigger if there are points outside of the domain, can't it?   
  
In that case this `assert` does not adhere to the API set out in the ReferenceManual for `CCTK InterpGridArrays` which documents `CCTK ERROR INTERP POINT OUTSIDE` as a possible return value and does not indicate that the code would abort.